pkg/providers/amifamily/ami.go

```diff
@@ -156,6 +156,7 @@ func (p *DefaultProvider) amis(ctx context.Context, queries []DescribeImageQuery
 		return append(AMIs{}, images.(AMIs)...), nil
 	}
 	images := map[uint64]AMI{}
+	currentTime := time.Now().Unix()
```

Comment:
Do we need this var? Can just do `time.Now().Unix()` where it's needed?

Reply:
Addressed